sqlalchemy-stubs/orm/session.pyi

```diff
@@ -45,6 +45,8 @@ class Session(_SessionClassMethods):
                  binds: Optional[Any] = ..., extension: Optional[Any] = ...,
                  info: Optional[Any] = ..., query_cls: Any = ...) -> None: ...
     connection_callable: Any = ...
+    def __enter__(self): ...
```

Comment:
The return type annotation is missing.

Reply:
This was copied from the `SessionTransaction` above. I matched the style there.

Reply:
```suggestion
    def __enter__(self) -> Session: ...
```

Reply:
This one is actually pretty important, as it tells Pylance/Pylint what type session is in

```
with Session() as session: